src/analyzer/plugins/analyzerqueenmarybeats.cpp

```diff
@@ -70,13 +70,14 @@ bool AnalyzerQueenMaryBeats::finalize() {
     }
 
     std::vector<double> df;
-    std::vector<double> beatPeriod(nonZeroCount);
+    std::vector<double> beatPeriod;
     std::vector<double> tempi;
 
     df.reserve(nonZeroCount);
-
-    for (int i = 0; i < nonZeroCount; ++i) {
+    // no idea why skip first 2 results, but that's how vamp does and works best this way
```

Comment:
This was changed by @daschuer in 67d8ecc63db4b8dd568b9ba1aff60f648ce93375.
And then seems like this logic re-introduces something similar: 944702c12961cef56879e5e2e2edbe6a2e03cc36

If we reintroduce this I think we should remove the added logic below.

Reply:
removed @daschuer patch for now as it did not affect the results, and I learned that if there is two ways to do the same thing I should stick with the simpler.
But I think we should hear from him, as he probably had a reason for doing so